redis/src/types.rs

```diff
@@ -2354,6 +2354,12 @@ impl FromRedisValue for Value {
     }
 }
 
+impl<T: FromRedisValue> FromRedisValue for RedisResult<T> {
```

Comment:
why is this needed?

Reply:
(see comment below)

Reply:
It's basically needed to the thingy you mentioned below, otherwise it'll lead to this error:

```
error[E0277]: the trait bound `Result<T, types::RedisError>: FromRedisValue` is not satisfied
    --> redis/src/cmd.rs:468:38
     |
468  |             from_owned_redis_value::<(u64, Vec<RedisResult<T>>)>(rv)?
     |                                      ^^^^^^^^^^^^^^^^^^^^^^^^^^ the trait `FromRedisValue` is not implemented for `Result<T, types::RedisError>`, which is required by `(u64, Vec<Result<T, types::RedisError>>): FromRedisValue`
     |
```

I think by implementing this, if we have a `Vec<RedisResult>` , then when parsing a bulk (i.e. array) reply, it will fail on the individual items, otherwise all the elements will end up failing.

Reply:
Removed